exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/partition/PruneScanRule.java

```diff
@@ -166,16 +174,28 @@ protected void doOnMatch(RelOptRuleCall call, DrillFilterRel filterRel, DrillPro
     }
 
     if (partitionColumnBitSet.isEmpty()) {
-      logger.debug("No partition columns are projected from the scan..continue.");
+      logger.info("No partition columns are projected from the scan..continue. " +
+          "Total pruning elapsed time: {} ms", totalPruningTime.elapsed(TimeUnit.MILLISECONDS));
```

Comment:
When the pruning rule does early return due to non-applicability, the 'Total pruning elapsed time' message could be misleading since it did not actually do pruning.